src/js/blue-button/containers/BlueButtonApp.jsx

```diff
@@ -31,7 +31,13 @@ class BlueButtonApp extends React.Component {
     return (
       <RequiredLoginView authRequired={3} serviceRequired={"bluebutton"}>
         <AppContent>
-          {this.props.children}
+          <div className="bb-app">
+            <div className="row">
+              <div className="columns small-12">
```

Comment:
If we're wrapping all the pages with this, I think the `.bb-app .row` rule can be removed from the stylesheet and the `row` class can be removed from `DownloadPage`.